app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,107 +1,56 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
-import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
 
-import DeleteAlert from "./DeleteAlert";
-import NewNotePane from "./Pane/Create";
-import Table from "./Table";
+import { NOTES_CARD_DATA as notesData } from "./constants";
```

Comment:
Be careful now. If something is documented in the books, then you ought to follow it. If you feel that a point ought to be updated in the book then you need to lemme know.

If you agree with what's mentioned in the book and if you are not following that practice, then it's not acceptable.

Refer: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react#:~:text=Use%20upper%20snake%20case%20in%20constant%20name